V/VLFeat/build_tarballs.jl

```diff
@@ -0,0 +1,81 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "VLFeat"
+version = v"0.9.21"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/vlfeat/vlfeat.git", "2f6abbf13fe7ee052bc970480efaca9d6b0c195d"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd vlfeat
+
+atomic_patch -p1 ../patches/vlfeat-march.patch
+atomic_patch -p1 ../patches/vlfeat-glibc-limitation.patch
+atomic_patch -p1 ../patches/vlfeat-mingw-lowercase-windows-include.patch
+atomic_patch -p1 ../patches/vlfeat-windows.patch
+
+make_args=""
+
+if [[ $target != x86_64-* ]]; then
+    make_args+=" \
```

Comment:
I believe string concatenation with `+=` is a bashism, which is OK because we use bash anyway, but a nicer way to append flags is to have use an array (we have plenty of uses here), which is also a bashism, but again that's ok

Reply:
If I do
```bash
make_args=()
if [[ $target != x86_64-* ]]; then
    make_args+="DISABLE_AVX=1 "
    make_args+="DISABLE_SSE2=1 "
fi

# ...

make $make_args[@]
```
I only get a trailing `[@]` in the targets to `make`

```bash
make "$make_args[@]"
```
only yields the first argument of the array.

https://stackoverflow.com/questions/55316852/append-elements-to-an-array-in-bash

Reply:
Never mind :-) `${make_args[@]}`